lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -90,6 +92,27 @@ class DetailsRenderer {
   }
 
   /**
+   * @param {!DetailsRenderer.LinkDetailsJSON} details
+   * @return {!Element}
+   */
+  _renderLink(details) {
+    const allowedProtocols = ['https:', 'http:'];
```

Comment:
in `is-on-https` audit we have a `SECURE_SCHEMES` array that i'd love to use and then add 'http' to, as our starting point. :)

Reply:
I'm not necessarily sold on using secure schemes here, just because it's transported securely doesn't mean it won't be malicious :) but seems like you've already highlighted that haha

Reply:
there may come a time when the report needs to link to a web socket resource, but it's not today :P blob/data URLs are more plausible, but nothing needs them yet.

My vote is for adding what we need now (linking to snyk vulnerability reports) and add others as they become needed (they haven't been needed yet by `convertMarkdownLinkSnippets`, for instance).